content/04.network_applications.md

```diff
@@ -39,5 +39,27 @@
 
 #### Clinical applications
 
-1. Can mention EHR use and other related applications
-2. Mention Tiffany's work on private data embeddings
+Using knowledge graphs for clinical applications is in early stages of development and these graphs are used to improve patent healthcare.
+Typically, these graphs are constructed from electronic health records (EHR) where nodes represent patients, drugs and diseases and edges represent relationships such as a patient being perscribed a treatment or a patient being diagnosed with a disease [@pmid:26306276; @doi:10.1145/2110363.2110415; @arxiv:1707.05340; @doi:10.1186/s12859-015-0549-5].
+Tasks in this context range from improving patient diagnoses [@doi:10.1109/TKDE.2016.2605687;@arxiv:1709.06908] to recommending safer drugs for patients [@arxiv:1710.05980; @arxiv:1709.06908].
+We discuss approaches that project knowledge graphs into a low dimensional space to make clinical predictions.
+
+Early work in this field used translational models (Figure {@fig:unifying_techniques_overview} (b)) on knowledge graphs to make safe drug recommendations.
+Wang et al. [@arxiv:1710.05980] used a variant of the TransH [@raw:wangtransH] model to create a recommendation system for to recmmend safe drugs to patients.
+The authors constructed a disease-patient-drug network by integrating a patient-disease bipartite network with a patient-drug bipartite network.
+Every node in the graph was embedded while satisfying the following equation: $\textbf{h} - r \approx \textbf{t}$.
+Following the embedding step, the authors formulated their own similarity metric that selected drug combinations with a low number of interactions [@arxiv:1710.05980].
+Researchers in [@arxiv:1909.00672] had a similar variant of the TransH model on a medical knowledge graph, but they evaluated their model link prediction rather than patient recommendation.
```

Comment:
```suggestion
Researchers in [@arxiv:1909.00672] applied a similar variant of the TransH model to a medical knowledge graph and evaluated their model for link prediction rather than patient recommendation.
```